config.py

```
@@ -0,0 +1,112 @@
+    time_delta_file = os.path.join(directory, "time_delta.txt")
+    if os.path.exists(time_delta_file):
+        try:
+            with open(time_delta_file, "r") as f:
+                content = f.read().strip()
+                if content and content.isdigit():
+                    _config["time_delta"] = int(content)
+                    changed = True
+            os.remove(time_delta_file)
+            print("Migrated time_delta.txt to config.json")
+        except Exception:
+            pass
```

Ruff (0.15.17)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)